lib/dependencies.js

```diff
@@ -1,59 +1,68 @@
 "use strict";
 
-var container, dizzy;
+var container, moduleDefs, Dizzy;
 
-dizzy = require("dizzy");
-container = dizzy.container();
-
-// Providing container so we can inject other modules
-// we don't want to specifically have called out here.
-container.provide("container", container);
```

Comment:
We use container in the storage module to load what storage we are using, so currently S3. We would need this back so we can use call on the S3 module, or look at a different way to load the storage module we want to use.